src/main/java/com/idrsolutions/microservice/JPedalServlet.java

```diff
@@ -270,88 +273,90 @@ protected boolean validateRequest(final HttpServletRequest request, final HttpSe
         return true;
     }
 
-    private static void convertPDF(final Mode mode, final String userPdfFilePath, final String outputDirStr,
-                                   final String fileNameWithoutExt, final Map<String, String> paramMap, final ErrorTracker durationTracker) throws Exception {
+    private ProcessUtils.Result convertFile(final Map<String, String> conversionParams,
+        final String uuid, final String webappDirectory, final File inputPdf,
+        final File outputDir, final long maxDuration) {
+
+
+        final ArrayList<String> commandArgs = new ArrayList<>();
+        commandArgs.add("java");
+
+        final Properties properties = (Properties) getServletContext().getAttribute(BaseServletContextListener.KEY_PROPERTIES);
+        final int memoryLimit = Integer.parseInt(properties.getProperty(BaseServletContextListener.KEY_PROPERTY_CONVERSION_MEMORY));
 
+        final int remoteTrackerPort = Integer.parseInt((String) properties.get(BaseServletContextListener.KEY_PROPERTY_REMOTE_TRACKING_PORT));
+
+        final float scaling = (conversionParams.containsKey("scaling") ? Float.parseFloat(conversionParams.get("scaling")) * 1.52f : 1.52f);
+        if (memoryLimit > 0) {
+            commandArgs.add("-Xmx" + memoryLimit + "M");
+        }
+
+        //Set settings
+        commandArgs.add("-Dcom.idrsolutions.remoteTracker.port=" + remoteTrackerPort);
+        commandArgs.add("-Dcom.idrsolutions.remoteTracker.uuid=" + uuid);
+
```

Comment:
Do we need to pass the settings from `conversionParam` here ([buildvu example](https://github.com/idrsolutions/buildvu-microservice-example/pull/71/files#diff-4e71fa4ee75caea8eae526fd78fb85013a3f57696e03d289d9c8e38e7120afdeR218-R224)) such as the password?